appinventor/docs/reference/components/sensors.html

```diff
@@ -662,43 +662,69 @@ <h2 id="LocationSensor">LocationSensor</h2>
 <p>Non-visible component providing location information, including longitude, latitude, altitude (if supported by the device), speed (if supported by the device), and address.  This can also perform "geocoding", converting a given address (not necessarily the current one) to a latitude (with the <code>LatitudeFromAddress</code> method) and a longitude (with the <code>LongitudeFromAddress</code> method).</p>
 <p>In order to function, the component must have its <code>Enabled</code> property set to True, and the device must have location sensing enabled through wireless networks or GPS satellites (if outdoors).</p>
 Location information might not be immediately available when an app starts.  You'll have to wait a short time for a location provider to be found and used, or wait for the OnLocationChanged event</p>
-
+<p>The emulator does not emmulate sensors. Code must be tested on a physical device.</p>
 <h3>Properties</h3>
 <dl>
   <dt><code><em>Accuracy</em></code></dt>
-  <dd></dd>
+  <dd>The LocationSensor will be able to locate the device with a varying degree of confidence, based on the quality of satellite, cell towers, and other data used to estimate location. The Accuracy value is the radius in meters around the sensor's detected location. The device is most likely located within this radius. More precise location detection will result in a
```

Comment:
So I'm not sure that the appropriate thing to do here is to use multiple `<dd>` per `<dt>` element. I think that this semantically implies that these are alternate definitions for the term in the `<dt>`. Instead, we should keep a single `<dd>` element per `<dt>` and nest multiple paragraphs using `<p>` tags.

Reply:
Should be done.